src/common/pf_bg_worker.h

```diff
@@ -0,0 +1,57 @@
+/*********************************************************************
+ *        _       _         _
+ *  _ __ | |_  _ | |  __ _ | |__   ___
+ * | '__|| __|(_)| | / _` || '_ \ / __|
+ * | |   | |_  _ | || (_| || |_) |\__ \
+ * |_|    \__|(_)|_| \__,_||_.__/ |___/
+ *
+ * www.rt-labs.com
+ * Copyright 2021 rt-labs AB, Sweden.
+ *
+ * This software is dual-licensed under GPLv3 and a commercial
+ * license. See the file LICENSE.md distributed with this software for
+ * full license information.
+ ********************************************************************/
+
+#ifndef PF_BG_WORKER_H
+#define PF_BG_WORKER_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Background worker jobs
+ */
+typedef enum pf_bg_job
+{
+   BG_JOB_UPDATE_PORTS_STATUS = 0, /**< Update status for all ports  */
```

Comment:
Also enum content needs to start with PF. Maybe
`PF_BGJOB_UPDATE_PORTS_STATUS`

Reply:
OK